test/sysinfo.jl

```diff
@@ -7,6 +7,8 @@ sprint(Base.Sys.cpu_summary)
 @test Base.Sys.uptime() > 0
 Base.Sys.loadavg()
 
+@test !("fakename" ∈ keys(ENV))
```

Comment:
You could create a GUID with https://www.guidgenerator.com/ and use that as the string.